docker/docker-entrypoint.sh

```diff
@@ -2,6 +2,10 @@
 
 set -e
 
+if [ "$(ls -A /usr/local/share/ca-certificates/)" ]; then
+  update-ca-certificates1
```

Comment:
Is this a typo or the command name really ends with a `1`?
```suggestion
  update-ca-certificates
```

Reply:
Typo, though how the hell did that make it in there? I did test the changes before I made the commits.